components/molecules/ProfileInfo.test.js

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { axe, toHaveNoViolations } from 'jest-axe'
+import ProfileInfo from './ProfileInfo'
+
+expect.extend(toHaveNoViolations)
+
+describe('ProfileInfo', () => {
+  it('renders ProfileInfo', () => {
+    render(<ProfileInfo />)
+    const text = screen.getByText('Ontario')
```

Comment:
If you pass the values in as props you can manually set them in the test, ensuring that the test will work independent of the data in the component.